src/overlays/actors/ovl_En_Hy/z_en_hy.c

```diff
@@ -635,57 +778,58 @@ void func_80A70978(EnHy* this, GlobalContext* globalCtx) {
     this->unk_1E8.unk_18 = player->actor.world.pos;
 
     if (LINK_IS_ADULT) {
-        this->unk_1E8.unk_14 = D_80A724A8[this->actor.params & 0x7F].unk_8;
+        this->unk_1E8.unk_14 = sInit1Info[this->actor.params & 0x7F].unkValueAdult;
     } else {
-        this->unk_1E8.unk_14 = D_80A724A8[this->actor.params & 0x7F].unk_4;
+        this->unk_1E8.unk_14 = sInit1Info[this->actor.params & 0x7F].unkValueChild;
     }
 
-    func_80034A14(&this->actor, &this->unk_1E8, D_80A724A8[this->actor.params & 0x7F].unk_0, phi_a3);
+    func_80034A14(&this->actor, &this->unk_1E8, sInit1Info[this->actor.params & 0x7F].unkPresetIndex, phi_a3);
 
-    if (func_800343CC(globalCtx, &this->actor, &this->unk_1E8.unk_00, this->unk_25C, func_80A6F810, func_80A70058)) {
+    if (func_800343CC(globalCtx, &this->actor, &this->unk_1E8.unk_00, this->unkRange, func_80A6F810, func_80A70058)) {
         func_80A70834(this, globalCtx);
     }
 }
 
-s32 func_80A70AE4(EnHy* this, GlobalContext* globalCtx) {
+s32 EnHy_ShouldSpawn(EnHy* this, GlobalContext* globalCtx) {
     switch (globalCtx->sceneNum) {
         case SCENE_SPOT01:
-            if (((this->actor.params & 0x7F) != 9) && ((this->actor.params & 0x7F) != 10) &&
-                ((this->actor.params & 0x7F) != 12) && ((this->actor.params & 0x7F) != 2) &&
-                ((this->actor.params & 0x7F) != 7)) {
+            if (!((this->actor.params & 0x7F) == ENHY_TYPE_BOJ_9 || (this->actor.params & 0x7F) == ENHY_TYPE_BOJ_10 ||
+                  (this->actor.params & 0x7F) == ENHY_TYPE_BOJ_12 || (this->actor.params & 0x7F) == ENHY_TYPE_AHG_2 ||
+                  (this->actor.params & 0x7F) == ENHY_TYPE_BJI_7)) {
                 return true;
             } else if (!LINK_IS_ADULT) {
                 return true;
-            } else if (((this->actor.params & 0x7F) != 12) && IS_NIGHT) {
+            } else if ((this->actor.params & 0x7F) != ENHY_TYPE_BOJ_12 && IS_NIGHT) {
```

Comment:
Pretty minor but I personally think parens around conditions actually help readability, so removing them is a slight downgrade imo

Reply:
We should enforce some rule about this, I'm on the "no-parentheses" side (it is more readable without them to me, especially in large conditionals), but I would be fully okay with going the opposite for the sake of consistency

Reply:
I'm fine with enforcing it, but we would probably need to discuss it with everyone first.
Also fwiw the vast majority of the codebase should have parens atm, but I guess it's not fully consistent either